### DISCUSSION TITLE
Sensitivie Discord Moderator

### DISCUSSION THREAD

**Discussion by BoomCheeseBurge** [ldtteam/minecolonies#9831] [General]
Hi, I was asking for help in the discord server since i couldnt access my minecraft account due to Microsoft's pain the butt way of handling migration and I cant retrieve my account whether to Mojang and Microsoft. They just kept throwing me from one to the other.  Afterwards, I got banned by the moderator around a year ago. Could somebody please unban me? Thanks.

By they way please watch this video regarding my issue which many others are experiencing.

https://www.youtube.com/watch?v=rUFDRAEducI

**Comment by Thodor12** [comment]
Unfortunately, not having a legal account, no matter how the circumstances came to be, is not allowed on our server.
You have to contact a moderator with proof of account ownership to get back into the server.

Alternatively you can still join back into the server, with a muted role (new policy we have) until you have a legitimate account, you can also DM a moderator for that if you want.

**Reply by BoomCheeseBurge** [reply]
I would love that if that is possible. May I know how I can DM a moderator to do that? Thank you. (P.S. I am still saving up to buy Minecraft and/or until Mojang/Microsoft can provide better services for people to retrieve back their account. I know a good service for account retrieval when I see one like Steam, for example).

**Reply by Thodor12** [reply]
Contact `motionlesstrain` or `shindria.` on Discord

**Reply by Raycoms** [reply]
Under our new policy, you are allowed to rejoin but will be unable to chat unless ownership is proven. Please leave your discord name here.

**Reply by BoomCheeseBurge** [reply]
Thank you, Thodor.

**Reply by BoomCheeseBurge** [reply]
Thank you, Raycoms. I understand. My discord name is "boomcheeseburger" without the quotation marks.

**Reply by MotionlessTrain** [reply]
You can join [discord](https://discord.minecolonies.com) again

**Reply by BoomCheeseBurge** [reply]
Apologies, when I opened the link I received the following error.

![image](https://github.com/ldtteam/minecolonies/assets/80831628/68f57988-b6e1-4280-a164-ac6295ebc2dc)

**Reply by MotionlessTrain** [reply]
Sorry, typo in url. It should work now

**Reply by BoomCheeseBurge** [reply]
No worries, thank you for approving my appeal to join back to the server.
